### PR TITLE
Update cardano's data with some well-known projects / organizations.

### PR DESCRIPTION
**Pull request by KtorZ**
Cheers ❤️

### CODE REVIEW

File: data/ecosystems/c/cardano.toml [hunk 1]
```diff
@@ -407,12 +512,6 @@ url = "https://github.com/CardanoUmbrella/awesome-cardano"
 [[repo]]
 url = "https://github.com/cffls/pycardano"
 
-[[repo]]
-url = "https://github.com/cloudstruct/go-cardano-submit-api"
-
-[[repo]]
-url = "https://github.com/cloudstruct/go-ouroboros-network"
```
Comment (eherrerosj) [inline data/ecosystems/c/cardano.toml:414]: why do you propose to remove these 2 repos?

**Review by eherrerosj — CHANGES_REQUESTED** [review]
Overall looks good! I would like to know why you are proposing to remove https://github.com/cloudstruct/go-cardano-submit-api and https://github.com/cloudstruct/go-ouroboros-network from Cardano taxonomy

File: data/ecosystems/c/cardano.toml (hunk 1 above)
Comment (KtorZ) [reply data/ecosystems/c/cardano.toml:414]: CloudStruct has become BlinkLabs, so I added the BlinkLabs equivalent instead 👍

Comment (KtorZ) [reply data/ecosystems/c/cardano.toml:414]: More specifically: 

| Old | New | 
| --- | --- | 
| https://github.com/cloudstruct/go-ouroboros-network | https://github.com/blinklabs-io/gouroboros |
| https://github.com/cloudstruct/go-cardano-submit-api | https://github.com/blinklabs-io/tx-submit-api |

You'll see that Github even redirects you properly if you click the old links.

**Review by eherrerosj — APPROVED** [review]
LGTM. Thanks for the contribution. Merging now

**Pull request merged**